support-frontend/assets/pages/[countryGroupId]/components/thankYouComponent.tsx

```diff
@@ -214,6 +216,11 @@ export function ThankYouComponent({
 		abParticipations.newspaperArchiveBenefit ?? '',
 	);
 
+	// Clarify Guardian Ad-lite thankyou page states
+	const isRegisteredAndSignedIn = !isNewAccount && isSignedIn;
+	const isRegisteredAndNotSignedIn = !isNewAccount && !isSignedIn;
+	const isNotRegistered = isNewAccount;
```

Comment:
Should we rename the variable instead of re-assigning it?

Reply:
Yes, good point. Reassignment removed and renamed.